application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -182,7 +163,16 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
                     getKeyByValue(VC_APPLICATION_TO_ID, applicationId).orElse("an activity");
         }
 
-        handleSubcommand(event, voiceChannel, applicationId, maxUses, maxAge, applicationName);
+        OptionMapping maxUsesOption = event.getOption(MAX_USES_OPTION);
+        OptionMapping maxAgeOption = event.getOption(MAX_AGE_OPTION);
+
+        if (maxUsesOption == null || maxAgeOption == null) {
+            handleSubcommand(event, voiceChannel, applicationId, 0, 86400, applicationName);
+        } else {
+            handleSubcommand(event, voiceChannel, applicationId,
+                    getOptionAsInteger(maxUsesOption).getAsInt(),
+                    getOptionAsInteger(maxAgeOption).getAsInt(), applicationName);
```

Comment:
- The If doesn't really make sense, if just one of the 2 is null you also ignore the other
 -  Whats the point of the OptionalInt? In else they will always exist.
 - Why not squash the if/else into an `getOptionalAsInteger(...).orElse(defaultValue)`?

Reply:
Also, magic numbers what does 0 / 86400 mean in this context?